src/Sharpliner/AzureDevOps/ConditionedExpressions/StaticVariableReference.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Sharpliner.AzureDevOps.ConditionedExpressions.Interfaces;
+
+namespace Sharpliner.AzureDevOps.ConditionedExpressions;
+
+public class StaticVariableReference : VariableReference, ICompileTimeExpression
```

Comment:
Can delete this now

Reply:
Thanks, will have a look at it. Please see my other comment about keeping this class though